website/blog/authors.yml

```diff
@@ -1,379 +1,348 @@
-  job_title: Senior Analytics Engineer
-  organization: dbt Labs
   image_url: /img/blog/authors/dconnors.jpeg
+  job_title: Senior Analytics Engineer
   links:
-    - url: https://github.com/dave-connors-3
-      icon: fa-github
-    - url: https://www.linkedin.com/in/daveconnors3/
-      icon: fa-linkedin
-
-ross_turk:
-  name: Ross Turk
-  job_title: VP Marketing
-  organization: Datakin
-  image_url: /img/blog/authors/ross-turk.png
-  links:
-    - url: https://mobile.twitter.com/rossturk
-      icon: fa-twitter
-    - url: https://github.com/rossturk
-      icon: fa-github
-
-sung_chung:
-  name: Sung Won Chung
-  job_title: Solutions Architect
+    - icon: fa-github
+      url: https://github.com/dave-connors-3
+    - icon: fa-linkedin
+      url: https://www.linkedin.com/in/daveconnors3/
+  name: Dave Connors
   organization: dbt Labs
-  image_url: /img/blog/authors/sung.jpeg
-  links:
-    - url: https://www.linkedin.com/in/sungwonchung1/
-      icon: fa-linkedin
-
-seth_rosen:
-  name: Seth Rosen
-  job_title: Co-Founder & CEO
-  organization: TopCoat Data
-  description: Seth Rosen is co-founder and CEO of TopCoat Data - a platform for helping organizations build analytical applications. Prior to founding TopCoat, Seth helped companies of all sizes build custom data apps on top of the modern data stack through his consultancy, Hashpath. When he’s not tweeting and thinking about data, he’s tweeting while parenting two toddlers.
-  image_url: /img/blog/authors/seth-rosen.jpeg
+david_krevitt:
+  description: Laziness is a virtue
+  image_url: /img/blog/authors/dkrevitt.jpg
+  job_title: Marketing
   links:
-    - url: https://twitter.com/sethrosen
-      icon: fa-twitter
-    - url: https://www.linkedin.com/in/sdrosen/
-      icon: fa-linkedin
-
+    - icon: fa-twitter
+      url: https://twitter.com/dkrevitt
+  name: David Krevitt
+  organization: dbt Labs
 donny_flynn:
-  name: Donny Flynn
-  job_title: Customer Data Architect
-  organization: Census
   description: Donny Flynn is a customer data architect at Census - a reverse ETL product that helps companies operationalize analytics. Prior to joining Census, Donny built out the data stack at Owner and led a data team at Chiper. When not doing data work, he's most likely watching Chicago sports.
   image_url: /img/blog/authors/dflynn.jpeg
+  job_title: Customer Data Architect
+  links:
+    - icon: fa-twitter
+      url: https://twitter.com/donmandonguy
+    - icon: fa-linkedin
+      url: https://www.linkedin.com/in/donny-flynn-578149a4/
+  name: Donny Flynn
+  organization: Census
+doug_beatty:
+  image_url: /img/blog/authors/dbeatty.jpeg
+  job_title: Senior Developer Experience Advocate
+  name: Doug Beatty
+  organization: dbt Labs
+emily_riederer:
+  description: Emily is very smart.
+  image_url: /img/blog/authors/emily-riederer.jpeg
+  job_title: Senior Manager Analytics
+  name: Emily Riederer
+  organization: Capital One
+  links:
+    - icon: fa-linkedin
```

Comment:
Could we please add Twitter (https://www.twitter.com/emilyriederer) and GitHub (https://www.github.com/emilyriederer)? If there's a common FontAwesome icon for adding personal sites, I'd love to link that also (https://www.emilyriederer.com/) but I didn't see others in the file, so it's fine either way

Reply:
done! got a book: 
![image](https://user-images.githubusercontent.com/7335046/211726645-9b6e5775-0b5d-44e6-b8c9-84ea7036f46a.png)